src/web/next.config.js

```diff
@@ -55,16 +61,27 @@ envVarsToForward.forEach((envVar) =>
   console.info(`Using NEXT_PUBLIC_${envVar}=${process.env[`NEXT_PUBLIC_${envVar}`]}`)
 );
 
-const withMDX = require('@next/mdx')({
-  extension: /\.mdx?$/,
-});
-
-module.exports = withMDX({
-  pageExtensions: ['ts', 'tsx', 'md', 'mdx'],
-  poweredByHeader: false,
-  reactStrictMode: true,
-  trailingSlash: true,
-  future: {
-    webpack5: true,
+// Compose all plugins
+module.exports = withPlugins([
+  [withMDX],
+  [
+    withPWA,
+    {
+      pwa: {
+        disable: process.env.NODE_ENV === 'development',
+        dest: 'public',
+        publicExcludes: ['!google*.html'],
+        runtimeCaching,
+      },
+    },
+  ],
+  {
```

Comment:
Let's do it the same way they do it in https://www.npmjs.com/package/next-compose-plugins#nextconfiguration with a `nextConfig` object above, so it's clear what is the base config, and what is a plugin config.